conans/client/remote_manager.py

```diff
@@ -18,14 +19,18 @@
 from conans.util.files import mkdir, tar_extract
 
 
-class RemoteManager(object):
+class RemoteManager:
     """ Will handle the remotes to get recipes, packages etc """
-
     def __init__(self, cache, auth_manager):
         self._cache = cache
         self._auth_manager = auth_manager
         self._signer = PkgSignaturesPlugin(cache)
 
+    @staticmethod
+    def _git_remote(remote):
+        if remote.url.startswith("file:///"):  # FIXME: remote type?
```

Comment:
I like inventing these so that we do not need to follow any existing specifications 

```suggestion
        if remote.url.startswith("conandiskremote://"):
```

Plain and easy, even if it's not registered it's hard to see that conflicting nor are we supporting generic protocols like a browser

Reply:
Yes not even that, but ``conan remote add myremote <path> --is-conandisk-remote``

Reply:
I think that's a better approach 👍  the URL path encoding for windows and network paths are painful and since this is not doing any encoding. With my background the overloading of the protocol scheme is confusing 🙃 

Does it need a flag at that point? What about just letting python sort it out? `if path.exists() and path.is_dir(): else it's a URL` 🤔  I wouldn't be surprised if that was more feature complete too.

From UX these could both work? Is there a reason for disk to be optional?

```
conan remote add disk-remote /usr/home/prince/conan-center-index-look-a-like
conan remote add disk-remote C:\Users\Chris Mc\Documents\conan-center-index-look-a-like
conan remote add local-remote https://myserver.prince.co/api
```